README.md

```diff
@@ -13,3 +9,55 @@ Make sure that you have the latest versions of NodeJS and Visual Studio Code in
+
+## Time formatting
+
+The Angular Material TimePicker component has a default, culture-specific time format, but it can be custom formatted
+
+## Related links
+[Learn More about Angular TimePicker](https://www.syncfusion.com/angular-components/angular-timepicker?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Documentation](https://ej2.syncfusion.com/angular/documentation/timepicker/getting-started/?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-angular-timepicker-component?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/angular-ui-components?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Online example](https://ej2.syncfusion.com/angular/demos/#/bootstrap5/timepicker/default?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/angular-ui-components?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples)
+
+## About Syncfusion Angular Components
+
+Syncfusion's [Angular Components](https://www.syncfusion.com/angular-components?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples) library is the only suite that you will ever need to build an application since it contains over 75 high-performance, lightweight, modular, and responsive UI Components in a single package. In addition to TimePicker, we provide popular Angular Components such as [DataGrid](https://www.syncfusion.com/angular-components/angular-grid?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples), [Charts](https://www.syncfusion.com/angular-components/angular-charts?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples), [Scheduler](https://www.syncfusion.com/angular-components/angular-scheduler?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples), [Diagram](https://www.syncfusion.com/angular-components/angular-diagram?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples), and [Word Processor](https://www.syncfusion.com/angular-components/angular-word-processor?utm_source=github&utm_medium=listing&utm_campaign=angular-timepicker-github-samples).
```

Comment:
build an application since it contains over 80 high-performance.........

Reply:
over 80 high-performance added